.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Docker Publish
+
+on:
+  workflow_run:
+    workflows : ["Docker build"]
+    branches: ["master"]
+    types:
+      - completed
+  push:
+    tags:
+      - "v*.*.*"
```

Comment:
Can you comment on this:
Does this mean that it also runs when we add a tag `v1.3.27`?
And does it also run, if we add the tag somewhat later after the push?
(We currently don't use this kind of versioning, but please comment here how this would behave, I assume you've tried it out).